File: hyeyoung/week11/푸드파이트대회.java
class Solution {
    public String solution(int[] food) {
        String answer = "";
        String reverse = "";
        for(int i = 1; i < food.length; i++){           
            for(int j = 0; j < food[i]/2 ; j++){
                answer += i;
            }                
        }
        StringBuilder sb = new StringBuilder(answer);
        reverse = sb.reverse().toString();
        answer = answer + 0 + reverse;
        
        return answer;
    }
}

File: seoyeon/week11/푸드파이트 대회.py
def solution(food):
    answer = ''
    stack = []
    repeat = 0
    for i in range(1, len(food)) :
        if food[i] % 2 != 0 : 
            food[i] -= 1
        if food[i] % 2 == 0 :
            repeat = food[i] // 2
            while(repeat > 0) :
                answer += str(i)
                repeat -= 1
    answer += '0'
    stack = answer[:-1]
    for i in range(1, len(stack) + 1) :
        answer += stack[-i]
    return answer